Packs/ShadowxSOCAI/Integrations/ShadowxSOCAI/ShadowxSOCAI.py

```diff
@@ -0,0 +1,535 @@
+        out['TaskURL'] = ui_url
+        if task_id and 'TaskId' not in out:
+            out['TaskId'] = task_id
+
+    return_results(CommandResults(
+        outputs_prefix='ShadowxSOCAI.TaskSubmit',
+        outputs=out,
+        readable_output=f"ShadowX task submitted (cookie mode)\nURL: {ui_url}" if ui_url else 'ShadowX task submitted (cookie mode)',
+        raw_response=out
+    ))
+
+
+def shadowx_get_task_command():
+    """Fetch task details by ID in API mode (Bearer)."""
+    task_id = demisto.args().get('task_id')
+    if not task_id:
+        return_error('task_id is required')
+    base_url = (demisto.params().get('url') or '').rstrip('/')
+    api_key = demisto.params().get('api_key') or ''
+    if not api_key:
+        return_error('shadowx-get-task requires API Key mode (set API Key in instance).')
+
+    s = _new_session()
+    data = check_task_with_api_key(s, base_url, api_key, task_id)
+    if not data:
+        return_error('Task not found or API did not return JSON')
+
+    parsed = _parse_task_fields(data, task_id, base_url)
+    _maybe_set_incident_fields(parsed)
+
+    return_results(CommandResults(
+        outputs_prefix='ShadowxSOCAI.TaskResult',
+        outputs=parsed,
+        readable_output=tableToMarkdown('ShadowX Task', parsed),
+        raw_response=data
+    ))
+
+
+def shadowx_help_command():
+    demisto.results("""
+### ShadowX SOCAI — Help
+
+**Commands**
+1) `!shadowx-submit-task` - submit a log (API Key or cookie mode)
+   - In API mode, set instance paths:
+     - API Submission Path: `/Api/SecurityTasks/Create`
+     - API Check Path Format: `/Api/SecurityTasks/Details?taskID={task_id}`
+   - Use `wait_seconds` to poll and return parsed results on first run.
+2) `!shadowx-get-task task_id="<GUID>"` - fetch details later (API mode)
+
+Returned fields (and Incident custom fields when in an incident):
+TaskName → shadowxaitaskname
+AssignedUserName → shadowxaiassigneduser
+AIDriverName → shadowxaiaidriver
+PolicyName → shadowxaipolicyname
+Subject → shadowxaisubject
+SecurityLog → shadowxaisecuritylog
+SanitizedLog → shadowxaisanitizedlog
+Response → shadowxairesponse
+Recommendation → shadowxairecommendation
+Status → shadowxaistatus
+RiskSeverity → shadowxairiskseverity
+PredictionScore → shadowxaipredictionscore
+""")
+
+
+# ============================
+# Main    
+# ============================
+
+def main():
+    try:
+        command = demisto.command()
+        demisto.debug(f'Command called: {command}')
+        if command == 'test-module':
+            test_module()
+        elif command == 'shadowx-submit-task':
+            shadowx_submit_task_command()
+        elif command == 'shadowx-get-task':
+            shadowx_get_task_command()
+        elif command == 'shadowx-help':
+            shadowx_help_command()
+        else:
+            return_error(f'Command not implemented: {command}')
+    except Exception as e:
+        return_error(f'Unexpected error: {str(e)}')
```

Comment:
please change the string to be something like: f"Failed to execute {command} command.\nError:\n{str(e)}"